database/seeders/modules/laravel-basics.php

```diff
@@ -0,0 +1,295 @@
+                'url' => 'https://laravelpodcast.com/episodes/eloquent-with-jonathan-reinink',
+                'type' => Resource::AUDIO_TYPE,
+            ],
+            [
+                'name' => 'The Laravel Podcast - Migrations, Factories, and Seeding, with John Bonaccorsi',
+                'url' => 'https://laravelpodcast.com/episodes/migrations-factories-and-seeding-with-john-bonaccorsi',
+                'type' => Resource::AUDIO_TYPE,
+            ],
+            [
+                'name' => 'The Laravel Podcast - Authorization and Authentication, with Joseph Silber',
+                'url' => 'https://laravelpodcast.com/episodes/authorization-and-authentication-with-joseph-silber',
+                'type' => Resource::AUDIO_TYPE,
+            ],
+            [
+                'name' => 'The Laravel Podcast - Presets & Jetstream, with Michael Dyrynda',
+                'url' => 'https://laravelpodcast.com/episodes/presets-jetstream-with-michael-dyrynda',
+                'type' => Resource::AUDIO_TYPE,
+            ],
+            [
+                'name' => 'The Laravel Podcast - The Service Container, with Christoph Rumpel',
+                'url' => 'https://laravelpodcast.com/episodes/the-service-container-with-christoph-rumpel',
+                'type' => Resource::AUDIO_TYPE,
+            ],
+            [
+                'name' => 'The Laravel Podcast - Cache, Session, Middleware, & Request, with Samantha Geitz',
+                'url' => 'https://laravelpodcast.com/episodes/cache-session-middleware-request-with-samantha-geitz',
+                'type' => Resource::AUDIO_TYPE,
+            ],
+            [
+                'name' => 'The Laravel Podcast - Storage, with Frank de Jonge',
+                'url' => 'https://laravelpodcast.com/episodes/storage-with-frank-de-jonge',
+                'type' => Resource::AUDIO_TYPE,
+            ],
+            [
+                'name' => 'The Laravel Podcast - Mail and Notifications, with Wilbur Powery',
+                'url' => 'https://laravelpodcast.com/episodes/mail-and-notifications-with-wilbur-powery',
+                'type' => Resource::AUDIO_TYPE,
+            ],
+            [
+                'name' => 'The Laravel Podcast - Helpers & Collections, with Jacob Baker-Kretzmar',
+                'url' => 'https://laravelpodcast.com/episodes/helpers-collections-with-jacob-baker-kretzmar',
+                'type' => Resource::AUDIO_TYPE,
+            ],
+            [
+                'name' => 'The Laravel Podcast - Security, with Rizqi Djamaluddin',
+                'url' => 'https://laravelpodcast.com/episodes/security-with-rizqi-djamaluddin',
+                'type' => Resource::AUDIO_TYPE,
+            ],
+            [
+                'name' => 'The Laravel Podcast - Queues, with Mohamed Said',
+                'url' => 'https://laravelpodcast.com/episodes/queues-with-mohamed-said',
+                'type' => Resource::AUDIO_TYPE,
+            ],
```

Comment:
Proposal: This should be a part of a new intermediate or advanced module surrounding Laravel Core
```suggestion
```